test/index-server.test.ts

```diff
@@ -7,12 +7,14 @@ describe("Client", () => {
   // avoid mocking fetch so we can test real requests. So we need to make sure
   // to put fetch back to what it was.
   let oldFetch: typeof fetch;
+  const consoleErrorSpy = jest.spyOn(console, "error").mockImplementation();
```

Comment:
Nice. I was going to ask about this in my previous PR. Glad we're on the same page!